examples/benzeneselenol.cjson

```diff
@@ -0,0 +1,298 @@
+                      91.6830475 , 91.7382866  ,  91.8732551 ,
+                      92.2830586 , 93.4248483  ,  94.0950647 ,
+                      94.7018786 , 96.940015   ,  97.312811 ,
+                      98.1329621 , 98.6845369  ,  98.9256298 ,
+                      99.8720418 , 101.066077  ,  102.622296 ,
+                      103.009787 , 103.157000  ,  103.31918 ,
+                      103.851435 , 104.263959  ,  104.477569 ,
+                      105.093634 , 105.367653  ,  105.811743 ,
+                      107.081426 , 107.640348  ,  109.544873 ,
+                      109.565826 , 111.875528  ,  112.978133 ,
+                      113.701956 , 117.159435  ,  117.843257 ,
+                      118.734702 , 119.00355   ,  119.216071 ,
+                      120.684942 , 121.162229  ,  122.132043 ,
+                      122.363612 , 122.407422  ,  122.699673 ,
+                      125.047471 , 125.335367  ,  127.834189 ,
+                      127.949293 , 128.899787  ,  129.180064 ,
+                      131.980660 , 132.208419  ,  133.187757 ,
+                      133.498511 , 136.356795  ,  136.904016 ,
+                      141.854039 , 143.408897  ,  150.746991 ,
+                      154.337262 , 155.484494  ,  155.865181 ,
+                      157.824673 , 159.69219   ,  165.816929 ,
+                      167.200628 , 167.743223  ,  169.743531 ,
+                      173.049987 , 175.582006  ,  177.226934 ,
+                      189.293823 , 193.292264  ,  194.618547 ,
+                      195.905101 , 196.00143   ,  196.633006 ,
+                      197.166349 , 197.213697  ,  199.31115 ,
+                      200.459743 , 201.63718   ,  202.001268 ,
+                      202.475018 , 203.157208  ,  203.208365 ,
+                      203.941984 , 204.921866  ,  204.986629 ,
+                      207.840015 , 207.85743   ,  211.716549 ,
+                      212.785684 , 213.701619  ,  213.909786 ,
+                      215.158517 , 215.867645  ,  216.13078 ,
+                      216.716641 , 224.298549  ,  224.521682 ,
+                      225.348364 , 226.129875  ,  226.421581 ,
+                      228.027325 , 228.53264   ,  228.580805 ,
+                      229.018092 , 229.292927  ,  229.85974 ,
+                      230.386552 , 231.492423  ,  234.01002 ,
+                      235.202695 , 236.990755  ,  238.367651 ,
+                      251.942867 , 255.645792  ,  255.900763 ,
+                      261.883730 , 262.938716  ,  264.091662 ,
+                      267.652000 , 676.858985  ,  693.106359 ,
+                      698.696666 , 702.410748  ,  703.761521 ,
+                      710.502053 , 1422.37774  ,  7092.06056 ,
+                      7105.18108 , 7106.74111  ,  26153.3786 ]]
+        "symmetry":[[ u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A', u'A',
```

Comment:
```symmetry``` also has the double ```[``` and the ```u``` from Python indicating a unicode string. Both should be removed.